src/lib/session.ts

```diff
@@ -24,16 +23,18 @@ export const initialise: () => Promise<void> = async () => {
   try {
     sessionStore.update(state => ({ ...state, loading: true }))
 
-    walletauth.setup.debug({ enabled: true })
-
     // Get the initial WNFS appState
-    const initialAppState = await walletauth.app({
-      onAccountChange: newAppState => handleAppState(newAppState),
-      onDisconnect: () => disconnect(),
+    const program = await walletauth.program({
+      namespace: { creator: "Fission", name: "Walletauth Template" },
+
+      onAccountChange: handleProgram,
```

Comment:
`handleProgram` doesn't seem to be called here, unless I switch it to something like `onAccountChange: (p) => handleProgram(p)`, which is interesting because [this](https://github.com/fission-codes/webnative-walletauth/pull/12/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R101) looks like it should trigger it the other way too 🤔

Reply:
Oh javascript 😒 I'll change it to `(p) => handleProgram(p)` 👍